### ISSUE TITLE
Would be nice to have an alert about isn't running app, for example was closed accidentally

### ISSUE THREAD

**Issue by qwqatest** [Faifly/xDrip#198]
Example (If I'm not mistaking the alert is shown only once after 10 min when app was close):
![Screenshot at Oct 14 16-27-04](https://user-images.githubusercontent.com/47991168/95997948-03887300-0e3d-11eb-9739-4b89183e6f66.png)

**Comment by qwqatest** [comment]
Verified
